lib/src/drawings.dart

```diff
@@ -370,8 +346,8 @@ class IconDrawing extends GYWDrawing {
         scale: (data["scale"] as num? ?? 1.0).toDouble(),
       );
     } else {
-      return IconDrawing.custom(
-        icon,
+      return IconDrawing(
+        GYWIcon(name: icon, filename: icon, width: 48, height: 48),
```

Comment:
Instead of manually setting 48 as width and height here, shouldn't we have these values as default in the constructor ? What do you think ?

Reply:
Sure.